go.mod

```diff
@@ -210,7 +210,7 @@ require (
 	github.com/keybase/go-keychain v0.0.0-20190712205309-48d3d31d256d // indirect
 	github.com/klauspost/compress v1.13.6 // indirect
 	github.com/koron/go-ssdp v0.0.2 // indirect
-	github.com/leanovate/gopter v0.2.10-0.20210127095200-9abe2343507a // indirect
+	github.com/leanovate/gopter v0.2.10-0.20210127095200-9abe2343507a
```

Comment:
nit/ move up to direct block?

Reply:
hm go mod tidy doesn't change it though?

Reply:
Yeah not sure :shrug: I think it tries to respect existing placement but I don't know the exact rules